.gitignore

```diff
@@ -19,7 +19,6 @@ coverage
 packages/components/.yarn/install-state.gz
 packages/components/.yarn/cache
 packages/tokens/dist/
-packages/tokens/figma/
```

Comment:
Was this just removed for testing purposes? The intent for it was so only the [build job](https://github.com/department-of-veterans-affairs/va-mobile-library/actions/workflows/tokens-build.yml) would be updating the figma folder instead of a mix of PRs and automation.

Reply:
Reverted